packages/shared/src/graphql/posts.ts

```diff
@@ -42,6 +42,11 @@ export const isInternalReadType = (post: Post): boolean =>
 export const isSharedPostSquadPost = (post: Post): boolean =>
   post.sharedPost?.source.type === SourceType.Squad;
 
+export const isVideoPost = (post: Post): boolean =>
+  post?.type === PostType.VideoYouTube ||
+  (post?.type === PostType.Share &&
+    post?.sharedPost?.type === PostType.VideoYouTube);
+
```

Comment:
In the previous examples we sometimes only checked the first line, just wondering if there where examptions where it was needed to not verify the share?

Reply:
I don't think so, but I'll double check

Reply:
Because we use `||`, the second part would only be checked if the first is false. And I don't think there are any exemptions to not verify the share.